lambdas/status_reports/requirements.txt

```diff
@@ -0,0 +1,3 @@
+Jinja2==3.1.2
+aiobotocore==2.3.4
+botocore==1.24.21
```

Comment:
I'd put these ones to setup.py as dependencies and generate requirements.txt with [pip-compile](https://pip-tools.readthedocs.io/en/latest/).

Reply:
what for?

Reply:
To have indirect dependencies pinned.

Reply:
ok i'll have a look

Reply:
done (tho i used py3.8-based venv to compile requirements, bc it's a PITA to install 3.7 on my machine -- it requires x64 arch, i hope it'll work)

Reply:
Well, it seems to work, but I think it really should be generated with the same Python version (and actually the same arch). 
It doesn't look like installing Python is really hard (https://diewland.medium.com/how-to-install-python-3-7-on-macbook-m1-87c5b0fcb3b5) or maybe you could you Docker. If you don't want to spend time on it, I could regenerate it by myself.

Reply:
> If you don't want to spend time on it, I could regenerate it by myself.

well, yeah, you can, but going forward i think it's time we move to a newer runtime (3.9 or even 3.10 would be nice)

Reply:
Well I think we already discussed that we use 3.7 so we can easily move code between components. 3.10 is not even available as lambda runtime.

Reply:
> Well I think we already discussed that we use 3.7 so we can easily move code between components

i dont remember that discussion, but well, this makes *some* sense, tho idk about moving anything from lambdas to the client, for example, and getting stuck with some really old runtime is quite frustrating